templates/include/rbs/constants.h.erb

```diff
@@ -0,0 +1,19 @@
+#ifndef RBS__CONSTANTS_H
+#define RBS__CONSTANTS_H
+
+extern VALUE RBS;
+extern VALUE RBS_AST;
+extern VALUE RBS_AST_Declarations;
+extern VALUE RBS_AST_Directives;
+extern VALUE RBS_AST_Members;
+extern VALUE RBS_Types;
+extern VALUE RBS_Types_Bases;
+extern VALUE RBS_ParsingError;
+
+<%- nodes.each do |node| -%>
+extern VALUE <%= node.c_constant_name %>;
+<%- end -%>
+
+void rbs__init_constants();
```

Comment:
This is subtly incorrect: Prior to C23 (which we aren't using), this means "takes unspecified params", not "takes no parameters".

```suggestion
void rbs__init_constants(void);
```

https://stackoverflow.com/a/51080

Reply:
Kept the original definition, not sure if we want to change it right now?

Reply:
Yep, I'll keep it "as-is" in our first PR, and add this to our second